pulpanalytics/templates/pulpanalytics/index.html

```diff
@@ -160,6 +161,28 @@
     }
 });
 
+// Postgresql Version Chart
+const postgresql_version_element = document.getElementById("postgresql-version");
+const postgresql_version_chart = new Chart(postgresql_version_element, {
+    type: 'pie',
+    data: {
+        datasets: [{
+            label: "Colors",
+            data: {{ postgresql_versions_count|safe }},
```

Comment:
I came to the conclusion, that this mechanism only works by coincidence. `safe` is not a filter that produces JSON output. It just so happens that it looks like valid JSON for a subset of JSON serializable objects.
Abetter way is to use the `json_script` filter. Or maybe we should switch to jinja2 as a template engine.

Reply:
+1 to this, I fixed this in my next push.